### ISSUE TITLE
główny algorytm Active Learning

### ISSUE THREAD

**Issue by kpmrozowski** [Akoszewski/Active-learning-svm-zum#3]
1) initialize process by applying base learner to labeled training data set to obtain Classifier
2) assign the val set the rest of the 'unlabelled' training data
3) normalize data
4) WhileLoop:
  4.1) get validation probabilities
  4.2) select samples using a selection function
  4.3) normalization needs to be inversed and recalculated based on the new train and test set.
  4.4) get the uncertain samples from the validation set
  4.4) normalize again after creating the 'new' train/test sets
5) print final active learning accuracies
